docs/migrating/migrating_from_ez_publish.md

```diff
@@ -44,6 +44,11 @@ If you are coming directly from legacy (4.x), you need to follow the best practi
 
 See Upgrade documentation on how to perform the actual upgrade: [Upgrade (eZ Publish Platform page)](https://doc.ez.no/display/EZP/Upgrade)
 
+!!! caution "Avoid exception when migrating the database"
+
+    If you plan to migrate from from eZ Publish through eZ Publish Platform 5.4 to eZ Platform and further, an exception may occur when you try to migrate the database while it contains internal drafts of Landing Pages, because such drafts do not have an expected row in the `ezcontentobject_name` table. 
```

Comment:
```suggestion
    If you migrate from from eZ Publish through eZ Publish Platform 5.4 to eZ Platform and further, an exception may occur when you try to migrate the database if it contains internal drafts of Landing Pages.
    This can happen because such drafts do not have an expected row in the `ezcontentobject_name` table. 
```
an attempt to shorten a pretty long sentence